integreat_cms/static/src/js/forms/tinymce-init.ts

```diff
@@ -167,15 +168,15 @@ window.addEventListener("load", () => {
             element_format: "html",
             entity_encoding: "raw",
             setup: (editor: Editor) => {
-                addIcon(editor, tinymceConfig, "pin", "meta+alt+1");
-                addIcon(editor, tinymceConfig, "www", "meta+alt+2");
-                addIcon(editor, tinymceConfig, "email", "meta+alt+3");
-                addIcon(editor, tinymceConfig, "call", "meta+alt+4");
-                addIcon(editor, tinymceConfig, "clock", "meta+alt+5");
-                addIcon(editor, tinymceConfig, "idea", "meta+alt+6");
-                addIcon(editor, tinymceConfig, "group", "meta+alt+7");
-                addIcon(editor, tinymceConfig, "contact", "meta+alt+8");
-                addIcon(editor, tinymceConfig, "speech", "meta+alt+9");
+                addIcon(editor, tinymceConfig, "pin", "meta+alt+1", "");
```

Comment:
I'm not entirely sure about the empty strings. Maybe there is a better way to solve this? But maybe someone else has a specific suggestion for this

Reply:
@JoeyStk it expects one more argument, and when I add related alt texts, it will be shown on dropdown options like here ( see `location` )
![image](https://github.com/digitalfabrik/integreat-cms/assets/77638520/d6167bad-b403-46de-884b-750d0775c3cc)